lib/bots/syncbot.ts

```diff
@@ -133,6 +134,11 @@ export class SyncBot extends ProcBot {
 						.return();
 					}
 				});
+				*/
```

Comment:
We should just delete this code entirely, if we want it to be disabled. We've got it in git anyway, and it'll make everything much clearer.